app/packages/core/src/components/Grid/Grid.tsx

```diff
@@ -50,13 +50,17 @@ function Grid() {
   const getFontSize = useFontSize(id);
 
   const spotlight = useMemo(() => {
+    records;
```

Comment:
what's the purpose of these two lines?

Reply:
Just to turn off eslint warnings. `records` can be removed, though